### PR TITLE
Kubeify Designator

### PR DESCRIPTION
**Pull request by zwolf**
Jenkinsfile and staging deployment template. Secrets are loaded.

### CODE REVIEW

File: kubernetes/deployment-staging.tmpl [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                  name: designator-staging-env
+                  key: DESIGNATOR_AUTH_PASSWORD
+          ports:
+            - containerPort: 80
+      volumes:
+        - name: static-assets
+          emptyDir: {}
+        - name: designator-nginx-config
+          configMap:
+            name: designator-nginx-conf
```
Comment (adammcmaster) [inline kubernetes/deployment-staging.tmpl]: Does this ConfigMap need to be added to this file?

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  options {
+    disableConcurrentBuilds()
+  }
+
+  stages {
+    stage('Build Docker image') {
+      agent any
+      steps {
+        script {
+          def dockerRepoName = 'zooniverse/cellect_ex'
```
Comment (adammcmaster) [inline Jenkinsfile]: This might be a good opportunity to change the repo name to designator.

File: kubernetes/deployment-staging.tmpl (hunk 1 above)
Comment (zwolf) [reply kubernetes/deployment-staging.tmpl]: Nope, missed this one (template is based on Caesar's)

File: Jenkinsfile (hunk 2 above)
Comment (zwolf) [reply Jenkinsfile]: I had that thought too but I wasn't certain what other downstream effects there might be.

**Pull request merged**